### DISCUSSION TITLE
Winding routes everywhere.

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#243] [Q&A] [answered]
If you really only see 👀 more and more curves anywhere, perhaps you should think about what you just consumed. 
[Winding routes](https://mranavigationnext.app/windingroutes/)

**Comment by devemux86** [comment, marked as answer]
We need to understand that the curvy / winding / twisty routes are essentially routes that avoid the main roads.

The remaining roads to use are not straight. The images and descriptions on these routing websites prove this.

---

But curvy routing is now possible with the new BRouter profile and options inside Cruiser and works **offline**!
Also, a new Cruiser profile is in the works with more surprise options to fine-tune your routes even better.

If the name is so important, can we add some (curvy or winding) presets that set automatically the BRouter options?

**Comment by Leeb64** [comment]
The trick is to avoid main roads only outside of Villages and Towns.
Villages and Towns should be avoided or driven through as quickly as possible...

**Reply by devemux86** [reply]
> Villages and Towns should be avoided

This is now available:
- https://github.com/devemux86/cruiser/discussions/248
